homeassistant/components/sensor/deconz.py

```diff
@@ -26,13 +25,13 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     if discovery_info is None:
         return
 
-    from pydeconz.sensor import DECONZ_SENSOR
+    from pydeconz.sensor import DECONZ_SENSOR, SWITCH
```

Comment:
To avoid confusion, I would suggest changing this to `…, SWITCH as DECONZ_SWITCH` here, since the term `switch` in HA refers to actuators, not sensors.

Reply:
Sure